.github/workflows/release.yml

```diff
@@ -39,6 +39,16 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - uses: elastic/apm-pipeline-library/.github/actions/github-token@main
```

Comment:
I saw this being done [in here](https://github.com/elastic/apm-agent-java/pull/3178/files) and, since the use case is similar for this repo, I'd like to apply the same git auth process.